aws/setup.sh

```diff
@@ -0,0 +1,25 @@
+# Set up Kubernetes
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=mrocklin@gmail.com
+kubectl --namespace kube-system create sa tiller
+kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+helm init --service-account tiller
+kubectl --namespace=kube-system patch deployment tiller-deploy --type=json --patch='[{"op": "add", "path": "/spec/template/spec/containers/0/command", "value": ["/tiller", "--listen=localhost:44134"]}]'
+
+# Get Helm repositories
+helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
+helm repo update
+
+# Install JupyterHub and Dask on the cluster
+helm install jupyterhub/jupyterhub --version=v0.6.0-9701a90 --name=jupyter --namespace=pangeo -f jupyter-config.yaml
+helm install dask/dask --name=dask --namespace=pangeo -f dask-config.yaml
```

Comment:
@mrocklin - question about `dask-config.yaml`. In the GCP deployment, you have also referenced this file but it is not included in the repository: 

https://github.com/pangeo-data/pangeo/blob/10dc105bae0083968417a8bec2b293ee1e6b4e2a/gce/setup.sh#L16-L18

Conversely, `jupyter-config.yaml` is included: https://github.com/pangeo-data/pangeo/blob/master/gce/jupyter-config.yaml

Any chance this was omitted from your commit/PR by accident?

Reply:
This file has been removed.  This line should be as well.